py/emitnative.c

```diff
@@ -287,12 +287,27 @@ struct _emit_t {
 #define ASM_CLR_REG(state, rd) ASM_XOR_REG_REG(state, rd, rd)
 #endif
 
+#if N_RV32
```

Comment:
This could be made slightly more general if combined with the above, eg:
```c
#ifdef REG_ZERO
// short cut macro for ASM_MOV_LOCAL_MP_OBJ_NULL
#else
// define REG_ZERO, ASM_CLR_REG, and long macro for ASM_MOV_LOCAL_MP_OBJ_NULL
#endif
```